applications/GeoMechanicsApplication/custom_processes/apply_phreatic_surface_pressure_table_process.hpp

```diff
@@ -25,19 +24,14 @@ class ApplyPhreaticSurfacePressureTableProcess : public ApplyConstantPhreaticSur
 {
 
 public:
-
     KRATOS_CLASS_POINTER_DEFINITION(ApplyPhreaticSurfacePressureTableProcess);
 
     /// Defining a table with double argument and result type as table type.
-    typedef Table<double,double> TableType;
-
-///----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+    typedef Table<double, double> TableType;
```

Comment:
I prefer the modern variant of `typedef` (which is more readable in my opinion):
```suggestion
    using TableType = Table<double, double>;
```

Reply:
Done